Library/Formula/terraform.rb

```diff
@@ -145,14 +63,12 @@ def install
     end
 
     cd terrapath do
-      system "go", "test", "./..."
+      terraform_files = `go list ./... | grep -v /vendor/`.strip.gsub("\n", " ")
+      system "go test #{terraform_files}"
```

Comment:
@DomT4 turns out you're suggested version does not work. I'll revert it to a working state.